### PR TITLE
Simplify data serialization through wrapper function

### PR DESCRIPTION
**Pull request by josegonzalez**
Previously, we would not surface *which* variable had errors serializing, resulting in an empty `variables` panel when any data was unserializable. With this change, variables that cannot be serialized are noted as such in the data array, as well as in the `error` key.

### CODE REVIEW

File: src/Routing/Filter/DebugBarFilter.php [hunk 1]
```diff
@@ -213,6 +207,40 @@ public function afterDispatch(Event $event)
...
+                try {
+                    serialize($value);
+                    $response[$key] = $value;
+                } catch (\Exception $e) {
+                    $response['error'] = $e->getMessage();
+                }
+            }
+        }
+
+        return serialize($response);
```
Comment (markstory) [inline src/Routing/Filter/DebugBarFilter.php]: Given that only the variables panel is the problem, couldn't this go there i stead of being applied to all panels?

Comment (josegonzalez) [inline src/Routing/Filter/DebugBarFilter.php]: Probably. How do you propose I handle multiple errors?

Comment (markstory) [inline src/Routing/Filter/DebugBarFilter.php]: Each problematic top level view var could have its content replaced with an inline error perhaps.

Comment (josegonzalez) [inline src/Routing/Filter/DebugBarFilter.php]: Alright, I'll take a look at that.

**Comment by josegonzalez** [comment]
Closed by #337 I think.

**Pull request closed without merging**